src/components/search/SearchedArtist.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+
+function SearchedArtist({ values }) {
+    const { id, name } = values;
+
+    
+    return (
+  
+        <Link to={`/${name}/${id}`}>
+            <h2>{name}</h2>
+            <h2>{id}</h2>
```

Comment:
```suggestion
            <h2>{id}</h2>
```
Is this for the user or the developer?